docs/using/running.md

```diff
@@ -107,6 +107,50 @@ docker rm notebook
 # notebook
 ```
 
+## Using the Podman CLI
+
```

Comment:
Let's add a note section with a brief description of current issues and then I think we are good to go.

Reply:
The example can be seen here: https://raw.githubusercontent.com/jupyter/docker-stacks/master/docs/maintaining/tasks.md

Reply:
I wrote two notes. See below.

I just realised that maybe want the notes to be at the start and not in the end of the section?

Reply:
Yes, I would prefer them to be in the beginning.

Reply:
I think you were right and I moved the notes later in the docs. Sorry for the trouble.